src/firefly/java/edu/caltech/ipac/firefly/server/query/lsst/LSSTQuery.java

```diff
@@ -421,4 +424,189 @@ else if ( classType.equalsIgnoreCase("NUMERIC") || classType.equalsIgnoreCase("D
 
     }
 
+    private static JSONObject jsonDataSet = LSSTQuery.getDataSet();
+    private static JSONObject getDataSet() {
+        JSONObject obj = new JSONObject();
+
+        // LSST dataset information for making table search request
+        String lsstDataSet = "{"
```

Comment:
this string mainly note the mission dependent metadata information for LSST search processor, such as ra/dec column name. It would be good to keep it in a file. Where is the good location for the file?